.github/workflows/release.yml

```diff
@@ -0,0 +1,45 @@
+# Creates a new bugfix release whenever a push is made to master.
+
+name: release
+
+# Controls when the action will run. 
+on:
+  push:
+    branches:
+      - master
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      
+      - name: Calculate next version
+        run: |
+          git pull
+          export CURRENT_VERSION=$(cat version.txt)
+          export THIS_VERSION=$(echo $CURRENT_VERSION | awk -F. -v OFS=. '{$NF++;print}')
+          echo "THIS_VERSION=$THIS_VERSION" >> $GITHUB_ENV
```

Comment:
Is there an advantage to doing this over using the actions output syntax?  https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#outputs

Reply:
Not that I'm aware of.
Is there an advantage to using the actions output syntax over the actions env syntax?

Reply:
No idea! :)  Either way looks like it works